.github/labeler.yml

```diff
@@ -6,7 +6,13 @@
 
 "Category: Build System":
 - build/*
+- win32/build/*
```

Comment:
This should probably be `win32/build/**`, same with `build`. The directories don't contain subdirectories though.